src/main/resources/swagger-current.yml

```diff
@@ -8,6 +8,7 @@ info:
 
     Version | Change
     ------- | ----------------------------------------------------------------
+    5.11.19 | All endpoints now prefixed with /ccsvc
```

Comment:
change the version above to keep it in sync

Reply:
Version updated